modules/core/04-channel/v2/keeper/msg_server.go

```diff
@@ -127,7 +127,7 @@ func (k *Keeper) RecvPacket(ctx context.Context, msg *channeltypesv2.MsgRecvPack
 			writeFn()
 		} else {
 			// Modify events in cached context to reflect unsuccessful acknowledgement
-			sdkCtx.EventManager().EmitEvents(convertToErrorEvents(cacheCtx.EventManager().Events()))
+			sdkCtx.EventManager().EmitEvents(internalerrors.ConvertToErrorEvents(cacheCtx.EventManager().Events()))
```

Comment:
think this is duplicate import, see `ibcerrors`

Reply:
oh wait, its not duplicate... should we just put it in `ibcerrors` none the less?

Reply:
I guess we can do that yeah, no harm

Reply:
oh, ibcerrors is exported, it's not under internal.